site/app/libraries/database/DatabaseQueries.php

```diff
@@ -136,6 +136,10 @@ public function updateSubmittyUserTimeZone(User $user, string $time_zone) {
         $this->submitty_db->query("update users set time_zone = ? where user_id = ?", [$time_zone, $user->getId()]);
         return $this->submitty_db->getRowCount();
     }
+    public function getSubmittyUserTimeZone(User $user) {
+        $this->submitty_db->query("SELECT time_zone FROM users WHERE user_id = ?", [$user->getId()]);
+        return ($this->submitty_db->getRowCount() > 0) ? $this->submitty_db->row()['time_zone'] : null;
+    }
```

Comment:
Instead of creating a separate query, is it possible to access this information through an existing `User` model?  Database queries are notoriously slow, so we want to minimize the total number of queries we make.